src/tests/utils/Processing.test.js

```diff
@@ -0,0 +1,88 @@
+import temp from 'temp';
+import { loadExternalDependenciesPlayground, readConfigPlayground, fillTank } from '../../utils/Processing';
+import { RawFSH } from 'fsh-sushi/dist/import/RawFSH';
+import { FHIRDefinitions } from 'fsh-sushi/dist/fhirdefs';
+import * as loadModule from '../../utils/Load';
+import 'fake-indexeddb/auto';
+
+describe('Processing', () => {
+  temp.track();
```

Comment:
I think this might have been left over from SUSHI, which tracks some temporary folders. Do you know if it's needed here?